themes/westerndevs/layout/_widget/home_page/recent_posts.ejs

```diff
@@ -18,10 +19,14 @@
           <% if (post.author && post.author.name ) { %>
           <B><%= post.author.name %></B><BR>
           <% } %>
-          <div class="ellipsis">
+          <div>
+	  <% if(cat) { %>
+			<div class="pull-right"><img src="images/microphone.png" alt="Podcast" title="Podcast" class="podcast-icon" /></div>
```

Comment:
How does our existing icon_podcasts.png image look here?

If it looks funny, consider using font-awesome instead of an image:

`<i class="fa-microphone" />`